### PR TITLE
press-kit: add press contacts and info

### PR DESCRIPTION
**Pull request by erciccione**
Added press contact and info as proposed in #1486. Suggestions are welcome.

The page still needs a lot of work, but for now i only added the new sections on top of what we already have.

preview of the entire page:

Screenshot 2021-07-12 at 14-29-16  Press Kit

### CODE REVIEW

File: _i18n/en.yml [hunk 1]
```diff
@@ -282,6 +282,18 @@ press-kit:
...
   mrllogo_para: The logo of the Monero Research Lab in SVG format.
+  contacts: Contacts
+  contactsp: There are multiple ways to contact the community. Depending by what information you are looking for, you might want to contact a workgroup, single members of the community, or the Core Team.
+  contactgen: "For press inquiries:"
+  contactwg: "To contact a specific Monero workgroup:"
+  contactcomm: "To contact the many subcommunities:"
+  whymonero: Why Monero?
+  whycommunity: Monero has one of the largest communities of any open-source project. Monero researches, audits, and implements critical security and privacy technologies, which are user by millions worldwide.
+  whycommunity1: Monero is trusted for more private transactions than all other cryptocurrencies combined.
+  whyconf: Monero is the only cryptocurrency project to be represented by its own village at Defcon, Grayhap and the CCC (Chaos Computer Club). Monero also hosts the Monero Konferenco, an academic conference, annually.
```
Comment (SamsungGalaxyPlayer) [inline _i18n/en.yml:293]: Replace with the following please:

The Monero community has substantial involvement at Defcon, Grayhat and the CCC (Chaos Computer Club). The Monero Konferenco, an academic conference, is hosted annually.

**Review by SamsungGalaxyPlayer — CHANGES_REQUESTED** [review]
Improve accuracy and "Grayhat"

**Comment by SamsungGalaxyPlayer** [comment]
image

File: _i18n/en.yml (hunk 1 above)
Comment (erciccione) [reply _i18n/en.yml]: Changed

**Review by SamsungGalaxyPlayer — APPROVED** [review]

**Review by selsta — APPROVED** [review]

**Pull request merged**